### PR TITLE
fix some deprecate warnings for v0.6

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

File: test/testCompareVariablesFactors.jl [hunk 1]
```diff
@@ -90,8 +90,8 @@ addFactor!(fg, [:x1;:l1], LinearConditional(Rayleigh()))
 
 sfg = buildSubgraphFromLabels!(fg, [:x0;:x1])
 
-
-@test compareFactorGraphs(fg, sfg, skip=[:labelDict;:addHistory;:logpath])
+#FIXME JT - this doesn't make sense to pass?
+# @test compareFactorGraphs(fg, sfg, skip=[:labelDict;:addHistory;:logpath])
```
Comment (Affie) [inline test/testCompareVariablesFactors.jl]: this doesn't make sense to pass

File: src/FGOSUtils.jl [hunk 2]
```diff
@@ -25,8 +25,9 @@ Related
```
Comment (dehann) [inline src/FGOSUtils.jl]: should we delete the docs here too?

File: src/JunctionTree.jl [hunk 3]
```diff
@@ -1633,7 +1633,7 @@ end
 
 Return the variable order stored in a tree object.
 """
-getVariableOrder(treel::AbstractBayesTree)::Vector{Symbol} = treel.variableOrder
+DistributedFactorGraphs.getVariableOrder(treel::AbstractBayesTree)::Vector{Symbol} = treel.variableOrder
```
Comment (dehann) [inline src/JunctionTree.jl]: tree not defined in DFG, is it?  Or is this a function name override issue?

If so, we should rather import the name from DFG and override here for dispatch.  I'm a little confused on why calling DFG explicitly here is the right thing to do.

File: test/testCompareVariablesFactors.jl (hunk 1 above)
Comment (dehann) [reply test/testCompareVariablesFactors.jl]: okay, that sounds fine

**Pull request merged**